projects/plugins/jetpack/extensions/shared/components/audio-player/index.jsx

```diff
@@ -72,9 +73,31 @@ function AudioPlayer( {
 		speak( __( 'Paused', 'jetpack' ), 'assertive' );
 	};
 
+	// Hacky patch to MediaElementPlayer to avoid the null reference error that can happen as the
+	// player is mounted and unmounted.
 	useEffect( () => {
+		if ( ! MediaElementPlayer.prototype._setResponsiveMode ) {
+			MediaElementPlayer.prototype._setResponsiveMode =
+				MediaElementPlayer.prototype.setResponsiveMode;
+			MediaElementPlayer.prototype.setResponsiveMode = function () {
+				const t = this;
+				if ( t.getElement( t.container ).parentNode ) {
+					t._setResponsiveMode();
+				}
+			};
+		}
+	}, [] );
+
+	useEffect( () => {
+		const container = containerRef.current;
+		const audioEl = document.createElement( 'audio' );
+		audioEl.src = trackSource;
+
+		// Insert player into the DOM.
+		container.appendChild( audioEl );
+		const mediaElement = new MediaElementPlayer( audioEl, meJsSettings );
+		audioRef.current = mediaElement.domNode;
```

Comment:
it seems that we could reference the audio element using the `audioEl` var defined above.

Reply:
We could, but MediaElement makes changes to the audio element, so the thinking was to update it with the most recent version according to MediaElement.

Reply:
In that case, we could select the element using the `containerRef` reference instead of creating a new one., as suggested above. 
Anyway, take it as a suggestion. Not a big deal.

Reply:
Yes that's true, we could remove the need for this reference. I'll have another look at it, as I didn't like having to keep references to two DOM elements.

Reply:
let me know if I can push some changes. Happy to help here 👍. Moving back to InProgress.